amundsen_application/static/css/_variables.scss

```diff
@@ -1,6 +1,7 @@
-// Bootstrap Default Values
-@import '~bootstrap-sass/assets/stylesheets/bootstrap/variables';
 // Amundsen Default Values
 @import 'variables-default';
 // Per-Client Custom Values
 @import 'variables-custom';
+// Bootstrap Default Values
+@import '~bootstrap-sass/assets/stylesheets/bootstrap/variables';
+
```

Comment:
Learned a bit about the nuances of `!default` values in SASS. 

Regular variables take the last definition.
```
$color: red;
$color: blue

$color => blue
```

Default variables take the first value.
```
$color: red !default;
$color: blue !default;

$color => red
```


This most closely mimics our current setup.
```
$color: red !default; // variables-default
$color: black; // variables-custom
$color: blue !default; // bootstrap/variables

$color => black
```